typed_python/PyPointerToInstance.cpp

```diff
@@ -114,7 +114,6 @@ PyObject* PyPointerToInstance::pointerGet(PyObject* o, PyObject* args) {
 //static
 PyObject* PyPointerToInstance::pointerCast(PyObject* o, PyObject* args) {
     PyInstance* self_w = (PyInstance*)o;
-    PointerTo* pointerT = (PointerTo*)extractTypeFrom(o->ob_type);
```

Comment:
@braxtonmckee is this variable unused because we forgot to use it? (The rest of the unused variables in this PR seem legitimate candidates for deletion, but I am unsure for this one).

Reply:
I think we can get rid of it. We don't need to know the source pointer type to do a cast.